src/health.c

```diff
@@ -369,7 +369,7 @@ void *health_main(void *ptr) {
         if(unlikely(check_if_resumed_from_suspention())) {
             apply_hibernation_delay = 1;
 
-            info("Postponing alarm checks for %ld seconds, because it seems that the system was just resumed from suspension."
+            info("Postponing alarm checks for %lld seconds, because it seems that the system was just resumed from suspension."
```

Comment:
All of these are compiler warning fixes about the size of `time_t`. You can reject them if you like.

Reply:
`time_t` is 64 bit on 64 bit systems, but I think they are 32 bit on 32 bit systems (the follow `size_t`). Can you check if `%zu` gives waning? If it does not, I think we will avoid warnings on 32 bit systems by using `%zu`.

Reply:
Update: Interesting. On my test machine, `ssize_t` is not the same size as `time_t`.
```
health.c:372:15: warning: format specifies type 'ssize_t' (aka 'long') but the argument
      has type 'time_t' (aka 'long long') [-Wformat]
            , hibernation_delay
              ^~~~~~~~~~~~~~~~~
./log.h:76:71: note: expanded from macro 'info'
#define info(args...)    info_int(__FILE__, __FUNCTION__, __LINE__, ##args)
                                                                      ^~~~
health.c:386:24: warning: format specifies type 'ssize_t' (aka 'long') but the argument
      has type 'time_t' (aka 'long long') [-Wformat]
                     , hibernation_delay
                       ^~~~~~~~~~~~~~~~~
./log.h:76:71: note: expanded from macro 'info'
#define info(args...)    info_int(__FILE__, __FUNCTION__, __LINE__, ##args)
                                                                      ^~~~
```

I think OpenBSD defines `time_t` as 64 bits, regardless of platform size, to prevent issues in 2038. On my 64-bit Linode (running 64-bit OpenBSD), `ssize_t` is `long`, but `time_t` is `long long`.